etc/v4l2loopback_setup.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ -z "$BASH_VERSION" ]] ; then
+    echo "Please run this script ($0) with bash, or execute it directly" >&2
+    exit 1
+fi
+
+if [[ "$(whoami)" != "root" ]]; then
+	echo "Please do run this script directly as root" >&2
+	exit 1
+fi
+
+if [[ ! "$(type apt-get)" ]]; then
+  echo "Unable to find APT package handling utility (apt-get)"
+  echo "Are you sure you're on a Debian-based Linux variant (e.g. Ubuntu)?"
+  exit 1
+fi
+
+install_dependencies() {
+  apt-get install --assume-yes kmod git make gcc libelf-dev
+}
+
+install_kernel_headers() {
+  echo "updating kernel module..."
+	apt-get --assume-yes update
+	apt-get --assume-yes dist-upgrade
```

Comment:
we should make clear that this script is for development purposes and that it will run a dist-upgrade which can be vary mutative for certain distros. we should be careful where this is running

Reply:
I'm working on removing the need for dist-upgrade now.

Reply:
sweet!

Reply:
Done